README.md

```diff
@@ -3,7 +3,7 @@ Right now this module is Android only, and only supports Interstitial Ads
 
 ## Run the Sample
 
-1. Copy the `ti.admob` folder to your `lib/` (Alloy) or your Resources (Classic) directory
+1. Copy the `ti-admob` folder to your `lib/` (Alloy) or your Resources (Classic) directory.
```

Comment:
Add a point with "Ensure you have the <transpile>true</transpile>` flag enabled in your tiapp.xml"